website/docs/installation/customize.mdx

```diff
@@ -28,6 +28,13 @@ oh-my-posh init pwsh --config 'C:/Users/Posh/jandedobbeleer.omp.json' | Invoke-E
 oh-my-posh init pwsh --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/jandedobbeleer.omp.json' | Invoke-Expression
 ```
 
+:::tip Non-ASCII file path
```

Comment:
This is only relevant for Windows PowerShell and nothing else so I wouldn't add it here.

Reply:
Make sense. I've moved it to faq.